std_execution.bs

```diff
@@ -5161,197 +5161,92 @@ are all well-formed.
-            
-            2. Otherwise, let `op_state2` be the result of `execution::connect(s, r)`. `execution::connect(s2, out_r)` returns an operation state `op_state` that stores `op_state2`. `execution::start(op_state)` is expression-equivalent to `execution::start(op_state2)`.
+                6. Let `sigs` be the unique set of types `[base_val_sigs..., base_err_sigs..., base_stopped_sigs..., out_sigs..., set_error_t(exception_ptr)]`.
 
-        3. Given some expression `e`, let `E` be `decltype((e))`, and let `S3` be `invoke_result_t<F>`. If either `sender<S, E>` or `sender<S3, E>` is `false`, the type of `tag_invoke(get_completion_signatures, s2, e)` shall be equivalent to `dependent_completion_signatures<E>`. Otherwise, let `As...` be the set of types in the <code><i>type-list</i></code> named by <code>value_types_of_t&lt;S, E, <i>set-value-signature</i>, <i>type-list</i>></code>, and let `Ws...` be the types in the <code><i>type-list</i></code> named by <code>error_types_of_t&lt;S, E, <i>type-list</i>></code>, where <code><i>set-value-signature</i></code> is the alias template:
-        
-            <pre highlight="c++">
-            template &lt;class... Ts>
-              using <i>set-value-signature</i> = set_value_t(Ts...);
-            </pre>
-
-            1. Let `Vs...` be the list of types in the <code><i>type-list</i></code> named by <code>value_types_of_t&lt;S3, E, <i>set-value-signature</i>, <i>type-list</i>></code>, and let `Es...` be the list of types in the <code><i>type-list</i></code> named by <code>error_types_of_t&lt;S3, E, <i>type-list</i>></code>.
+                7. The type of `get_completion_signatures(s2, e)` is equivalent to: `completion_signatures<sigs>`.
 
-            4. Let `Us...` be the set of unique types in `[As..., Vs...]` and let `Ws...` be the unique set of types in `[exception_ptr, Es...]`,.
 
-            5. If `completion_signatures_of_t<S3, E>::sends_stopped` is `true`, the type of `get_completion_signatures(s2, e)` is equivalent to:
-
-                <pre highlight="c++">
-                completion_signatures &ltUs..., set_error_t(Ws)..., set_stopped_t()>
-                </pre>
-
-                Otherwise:
-
-                <pre highlight="c++">
-                completion_signatures &ltUs..., set_error_t(Ws)...>
-                </pre>
 
-    If `execution::let_stopped(s, f)` does not return a sender that invokes `f` when `set_stopped` is called, and making its completion dependent on the completion of a sender returned by `f`, and propagates the other completion-signals sent by `s`, the behavior of calling `execution::let_stopped(s, f)` is undefined.
+    If <code><i>let-cpo</i>(s, f)</code> does not return a sender that invokes `f` when <code><i>set-cpo</i></code> is called, and making its completion dependent on the completion of a sender returned by `f`, and propagates the other completion-signals sent by `s`, the behavior of calling <code><i>let-cpo</i>(s, f)</code> is undefined.
```

Comment:
I know this was there prior, but I think we need to s/making/makes/ to make this make sense grammatically; can you do that as a drive-by fix?